pkg/vulnsrc/chainguard/chainguard.go

```diff
@@ -0,0 +1,121 @@
+package chainguard
+
+import (
+	"encoding/json"
+	"io"
+	"path/filepath"
+	"strings"
+
+	bolt "go.etcd.io/bbolt"
+	"golang.org/x/xerrors"
+
+	"github.com/aquasecurity/trivy-db/pkg/db"
+	"github.com/aquasecurity/trivy-db/pkg/types"
+	"github.com/aquasecurity/trivy-db/pkg/utils"
+	"github.com/aquasecurity/trivy-db/pkg/vulnsrc/vulnerability"
+)
+
+const (
+	chainguardDir = "chainguard"
+	distroName    = "chainguard"
+)
+
+var (
+	source = types.DataSource{
+		ID:   vulnerability.Chainguard,
+		Name: "Chainguard Secdb",
```

Comment:
nit: It is called "Chainguard Security Data" here. Which is the correct name for Chainguard security advisories?

Reply:
Sorry I forgot to put a link. We want to be consistent in what we call it across our projects.
https://github.com/aquasecurity/vuln-list-update/blob/c98ad9e9909ecb97de0f8777131c251023167c81/main.go#L245

Reply:
Pushed an update!